BaikalTest/input_maps.h

```diff
@@ -554,7 +554,7 @@ TEST_F(InputMapsTest, InputMap_Shuffle)
 {
     auto material = Baikal::UberV2Material::Create();
     auto color1 = Baikal::InputMap_ConstantFloat3::Create(float3(1.0f, 0.0f, 0.0f));
-    auto diffuse_color = Baikal::InputMap_Shuffle::Create(color1, {0, 0, 0, 0});
+    auto diffuse_color = Baikal::InputMap_Shuffle::Create(color1, {{0, 0, 0, 0}});
```

Comment:
Why you made this change?

Reply:
It seems to be a bug in clang! See [stackoverflow](https://stackoverflow.com/questions/31555584/why-is-clang-warning-suggest-braces-around-initialization-of-subobject-wmis).

Reply:
then just move this part out to std::array<uint32_t, 4> shuffle = {0, 0, 0, 0};

Reply:
Nothing has changed, i've got the same warning. Clang doesn't allow me to use initializer list with single braces anyway.